app/src/main/java/fr/free/nrw/commons/LicenseList.java

```diff
@@ -13,10 +13,18 @@
 import java.util.Locale;
 import java.util.Map;
 
+/**
+ * Represents LicenseList
```

Comment:
A list of licenses, actually :-)
Would you mind correcting?

Reply:
No problem I will change it